src/core/core-adapter.js

```diff
@@ -312,35 +313,36 @@ export default class CoreAdapter {
     const queryTrigger = this.getQueryTriggerForSearchApi(
       this.globalStorage.getState(StorageKeys.QUERY_TRIGGER)
     );
-    return this._searcher
-      .universalSearch(queryString, {
-        geolocation: this.globalStorage.getState(StorageKeys.GEOLOCATION),
+    return this._coreLibrary
+      .universalSearch({
+        query: queryString,
+        coordinates: this.globalStorage.getState(StorageKeys.GEOLOCATION),
         skipSpellCheck: this.globalStorage.getState('skipSpellCheck'),
         queryTrigger: queryTrigger,
         sessionTrackingEnabled: this.globalStorage.getState(StorageKeys.SESSIONS_OPT_IN).value,
         context: context,
-        referrerPageUrl: referrerPageUrl,
-        querySource: this.globalStorage.getState(StorageKeys.QUERY_SOURCE)
+        referrerPageUrl: referrerPageUrl
+        // querySource: this.globalStorage.getState(StorageKeys.QUERY_SOURCE)
```

Comment:
This is used for analytics to track whether or not queries come from the Overlay - so we'll want to add this back in eventually but I'm thinking it's not in the answers-core lib yet?

Reply:
It's not in the answers-core yet but I can put up a PR